modules/olm-installing-from-operatorhub-using-web-console.adoc

```diff
@@ -81,12 +81,12 @@ endif::[]
 .. For clusters on cloud providers with token authentication enabled:
 +
 --
-* If the cluster is in AWS STS mode, enter the Amazon Resource Name (ARN) of the AWS IAM role of your service account in the *role ARN* field.
+* If the cluster uses {aws-short} {sts-short} (*STS Mode* in the web console), enter the Amazon Resource Name (ARN) of the AWS IAM role of your service account in the *role ARN* field.
```

Comment:
Does the STS before Mode need to be {entra-first}?

Reply:
It would be `{sts-short}`, but the web console text is not tied to what we say in docs so I think it is safer to leave such instances verbatim (in case they update it to something else).

Reply:
Sounds good!